examples/tess/tess_binrates_func.jl

```diff
@@ -0,0 +1,949 @@
+## ExoplanetsSysSim/examples/dr25_gaia_fgk/dr25_binrates_func.jl
```

Comment:
Again, I didn't review this in detail.  Let me know if there's anythinng you changed that you'd like me to look at.